examples/ksonnet/components/10-configmaps.jsonnet

```diff
@@ -91,6 +91,7 @@ local sonobuoyConfigData = {
 local tolerations = [
     {
         key: "node-role.kubernetes.io/master",
+        operator: "Exists",
```

Comment:
So do you know why the original reason was failing to schedule on your master nodes?

Reply:
I cannot for the life of me remember. I went through the slack history and nothing triggered for me.

Very likely it was an error on my part (something dumb like forgetting to add tolerations at all) and removing `Exists` still worked in this case since we don't specify any value on the master taint.

After re-reading the documentation and experimenting, I'm very confident this is the change we want. 

The key piece from [here](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) is 

> A toleration “matches” a taint if the keys are the same and the effects are the same, and:
> * the operator is Exists (in which case no value should be specified), or
> * the operator is Equal and the values are equal
> * Operator defaults to Equal if not specified.